app/src/main/res/values-en-rGB/strings.xml

```diff
@@ -463,4 +463,4 @@
     <string name="pref_show_backdrop_description">Change background image to selected items</string>
     <string name="home_section_resume_book">Continue reading</string>
     <string name="lbl_shuffle_queue">Toggle Shuffle</string>
-</resources>
\ No newline at end of file
+</resources>
```

Comment:
Please revert the changes to this file and force push so it doesn't stay in the git history (this will break our translate tool)